scripts/export_quantized_model.py

```
@@ -0,0 +1,435 @@
+
+        # Prepare for QAT
+        qat_model = qat_trainer.prepare()
+
+        # Simple training loop
+        optimizer = torch.optim.Adam(qat_model.parameters(), lr=args.qat_lr)
+        criterion = torch.nn.MSELoss()
+
+        qat_model.train()
+        for epoch in range(args.qat_epochs):
+            total_loss = 0
+            for batch_idx, (batch,) in enumerate(train_loader):
```

Ruff (0.15.5)

[warning] Loop control variable `batch_idx` not used within loop body

Rename unused `batch_idx` to `_batch_idx`

(B007)